.editorconfig

```diff
@@ -17,4 +17,4 @@ indent_style = space
 indent_size = 4
 end_of_line = lf
 trim_trailing_whitespace = true
-max_line_length = 88
+max_line_length = 79
```

Comment:
Could we make it a nice round 80 or 85?

Reply:
pep8 advise max line length 79 char
but if you like you can make a suggestion to make it to 80 or 85.

Reply:
```suggestion
max_line_length = 80
```